scipy/signal/signaltools.py

```diff
@@ -92,9 +113,9 @@ def correlate(in1, in2, mode='full'):
     in1 : array_like
         First input.
     in2 : array_like
-        Second input. Should have the same number of dimensions as `in1`;
-        if sizes of `in1` and `in2` are not equal then `in1` has to be the
-        larger array.
+        Second input. Should have the same number of dimensions as `in1`.
+        If operating in 'valid' mode, either `in1` or `in2` must have
+        dimensions that are at least as large as the other.
```

Comment:
Nitpick: I think this (and the analogous portion of the other docstrings) would read slightly clearer phrased as: 
"must be at least as large as the other in all dimensions."